### DISCUSSION TITLE
There was an error parsing the JSON document. The document may not be well-formed.

### DISCUSSION THREAD

**Discussion by osde8info** [community/community#42594] [Projects and Issues]
### Select Topic Area

Bug

### Body

use 
windows 10 edge

goto
https://github.com/osde8info/vercel-nextjs-22/blob/main/package.json

keep pressing F5

sometimes it works and sometime you see the error

```
There was an error parsing the JSON document. The document may not be well-formed.
Unexpected token 'C', "Created fr"... is not valid JSON
Created from https://vercel.com/new
```

![image](https://user-images.githubusercontent.com/51541/209461629-98a4860a-b136-4f32-a570-a8d0aa4c9987.png)

![image](https://user-images.githubusercontent.com/51541/209461637-9fe120d9-0318-4dac-babc-0bb7d5a73547.png)

**Comment by narges1212** [comment]
remove the jsonview from the browser fixed my problem
